corona-warn-app/cwa-website
# Fix hotline link in May 12, 2021 blog (resolves #1796)

This PR resolves a bad link to the app hotline in May 12, 2021 blog (DE) from issue https://github.com/corona-warn-app/cwa-website/issues/1796.

It changes the hyperlink for the international app hotline number "+49 30 498 75401" in https://github.com/corona-warn-app/cwa-website/blob/master/blog/2021-05-12-cwa-2-2/index_de.md from non-working `https://www.coronawarn.app/de/blog/2021-05-12-corona-warn-app-version-2-2/+493049875401` to `tel:+493049875401` which will invoke a dialing response in an appropriate environment.

### Steps to verify

1. Open http://localhost:8000/de/blog/2021-05-12-corona-warn-app-version-2-2/ in a webbrowser
2. Scroll down to the paragraph starting "Falls ihnen ein Fehler auffällt ..."
3. Click on the hyperlink [+49 30 498 75401](tel:+493049875401)
4. Depending on the browser environment, you may see a pop-up like "Make a call from ..."

To verify in production open https://www.coronawarn.app/de/blog/2021-05-12-corona-warn-app-version-2-2/ with same steps as above.
